client/lomas_client/client.py

```diff
@@ -63,25 +66,62 @@ def __init__(self, url: str, user_name: str, dataset_name: str) -> None:
         self.headers["user-name"] = user_name
         self.dataset_name = dataset_name
 
+    def get_df_dtypes(self) -> Dict[str, str]:
+        """
+        Returns the pandas dictionary for the dataframe types of the dataset.
+
+        Raises:
+            ValueError: If metadata does not have the correct format or\
+                cannot be fetched.
+
+        Returns:
+            Dict[str, type]: The dtypes dictionary
+        """
+        metadata = self.get_dataset_metadata()
+        if metadata is None:
+            raise ValueError(
+                f"Could not get metadata for dataset {self.dataset_name}."
+            )
+        dtypes = {}
+
+        # Note if metadata was a pydantic model, this would be cleaner.
+        if not isinstance(metadata["columns"], dict):
+            raise ValueError("Metadata not in correct format.")
```

Comment:
Metadata is a pydantic model, you can check and enhance it at the bottom of collection_models.py and then probably copy it in the client (for now)

Reply:
Ah nice, so I should probably enhance the model with the info I added to the metadata (e.g. precision) for int and floating points

Reply:
it is very basic for now but I agree it would be nice to have it as complete as possible